rootfs/scheduler/__init__.py

```diff
@@ -1004,12 +1004,7 @@ def _scale_rc(self, namespace, name, desired):
             'version': rc['spec']['selector']['version']
         }
 
-        # Are there any pods running (and verified as ready) available?
-        pods = self._get_pods(namespace, labels=labels).json()['items']
-        current = 0
-        for pod in pods:
-            if self._pod_ready(pod):
-                current += 1
+        current = int(rc['spec']['replicas'])
```

Comment:
Correct but look at bb844717170d355a6e75ebe371a5efd090defd88 and 22feb4478f4a0c0b9c7e1d751448ba2a5dbd44cd which are the progression behind why `_scale_rc` is the way it is.

I just want to be very cautious about what we are doing in `_scale_rc`